roxie/ccd/ccdserver.cpp

```diff
@@ -12039,6 +12042,7 @@ class CRoxieServerIndexWriteActivity : public CRoxieServerInternalSinkActivity,
     {
         CRoxieServerActivity::start(parentExtractSize, parentExtract, paused);
         resolve();
+        duplicateKeyCount = 0;
```

Comment:
if inside a child query (highly unusual) this would reset the duplicate count.  That is correct for publishing with the file, but not for the accumulated statistic.